### PR TITLE
CLOUDP-70021: mongocli iam team(s) user(s) add

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70021

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

Atlas
```
./bin/mongocli iam team users add 5e4bc367c6b0f41bb9bbb178 5f4e89f73a9f5c0c8872ca8f --teamId 5f4f7e43a489de6dd3c0e463
User(s) added to the team.
```
Screenshot 2020-09-02 at 13 13 48

OM
```
./bin/mongocli iam team users add 5f4f3c760ed5b166b56b02cb --teamId 5f4f7e950ed5b166b56b34aa -P ops
User(s) added to the team.
```

### CODE REVIEW

File: internal/cli/iam/teams/users/add.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const addTemplate = `ID	FIRST NAME	LAST NAME	USERNAME	EMAIL{{range .}}
+{{.ID}}	{{.FirstName}}	{{.LastName}}	{{.Username}}	{{.EmailAddress}}{{end}}
```
Comment (gssbzn) [inline internal/cli/iam/teams/users/add.go]: for things like create/add/invite we usually just confirm, so we should say something like
`"{{len .}} User(s)  added to the team\n"` or something like that

File: internal/cli/iam/teams/users/add.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+				opts.init,
+				opts.InitOutput(cmd.OutOrStdout(), listTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.teamID, flag.TeamID, "", usage.TeamID)
```
Comment (gssbzn) [inline internal/cli/iam/teams/users/add.go:70]: I guess this should be required, right?

**Review by gssbzn — CHANGES_REQUESTED** [review]
A couple of small nits, also, any plans on adding e2e for teams any time soon?

File: internal/cli/iam/teams/users/add.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/iam/teams/users/add.go]: this has been already fixed

**Comment by andreaangiolillo** [comment]
I'd like to implement e2e tests once I have finished to implement all the commands in the milestone

**Review by gssbzn — APPROVED** [review]
LGTM don't wait too long for those e2e

**Pull request merged**